src/overlay021/ov21_021D5600.c

```diff
@@ -117,27 +119,27 @@ Strbuf* ov21_021D56BC (int param0, int param1, int param2, int param3)
         v2 = v0 * 1 + param2;
     }
  
-    return ov21_021D5724(v3, v2, param3);
+    return LoadMessage(v3, v2, param3);
 }
 
 static inline BOOL inline_ov21_021D5764_1 (int param0, int param1)
 {
-    if ((param0 > 493) && (param1 != 6)) {
+    if ((param0 > NATIONAL_DEX_COUNT) && (param1 != 6)) {
         return 0;
     }
 
     return 1;
 }
 
-static Strbuf* ov21_021D5724 (int param0, int param1, int param2)
+static Strbuf* LoadMessage (int bankID, int entryID, int heapID)
```

Comment:
This routine could do with some cleanup on variables names, while you're in here.

Reply:
Done